enacts/onset/maproom.py

```diff
@@ -608,7 +662,7 @@ def onset_tile(tz, tx, ty):
             dry_spell_length,
             0
         )
-        mymap_max = np.timedelta64((precip_tile["T"][-1] - precip_tile["T"][0]).values, 'D')
+        map_max = np.timedelta64((precip_tile["T"][-1] - precip_tile["T"][0]).values, 'D')
```

Comment:
I pushed a commit for that. However, there are other parts of the code that rely on T being ordered (grep for -1 to see the few instances). I am not sure what to think about assuming T being ordered or not in general. And in this specific case where onset and cessation algorithms are notoriously relying on time order by definition.

Reply:
So just put an assertion after maproom.py line 41 (the call to read_zarr_data) that verifies that T is in increasing order. Then the change you made here is unnecessary.

If the assertion ever fails, it will be obvious what's wrong, whereas if a dataset with out-of-order T sneaks in, the effects might be subtle and hard to debug.